src/main/java/io/confluent/copycat/jdbc/DataConverter.java

```diff
@@ -225,10 +222,10 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
       case Types.TIMESTAMP: {
         // Current approach uses UNIX time in milliseconds. May lose nanosecond precision
         // available in source type
-        if (nullableBuilder != null) {
-          nullableBuilder.longType().endUnion().noDefault();
+        if (optional) {
```

Comment:
Ah, I just mean giving the schema a name. So something like `builder.name("io.confluent.copycat.jdbc.Timestamp")`. But we can also just hold off until we have the logical type in Copycat, which will have a name set to something like `org.apache.kafka.copycat.Timestamp` anyway.